deployments/rds-s3/terraform/main.tf

```diff
@@ -119,54 +119,86 @@ module "eks_blueprints" {
   tags = local.tags
 }
 
+module "ebs_csi_driver_irsa" {
+  source  = "terraform-aws-modules/iam/aws//modules/iam-role-for-service-accounts-eks"
+  version = "~> 5.20"
+
+  role_name_prefix = "${local.cluster_name}-ebs-csi-driver-"
```

Comment:
please use region as well

do we need this module here or can we keep it in iaac/terraform/aws-infra and reuse it in other places?

Reply:
good idea, will move it there.

Reply:
done